### PR TITLE
Additional info section

### PR DESCRIPTION
**Pull request by polyccon**
Fixes:
#119 
#178 

Added:
* additional_info.hbs 
* additional_info.js
* endpoints in index.js
* middleware in app.js
* added additional_info section in database 
* added tests for testing the save and get section, and the get form queries
* added tests for post function with correct and wrong password and the logout button

### CODE REVIEW

File: src/views/progress.hbs [hunk 1]
```diff
@@ -51,6 +51,18 @@
   </div>
 </article>
 
+<article class="w-100 mw7 center bb b--black-10 pb4 pt4">
+  <div class="flex justify-between flex-wrap items-center w-80 margin-0-auto">
+    <div class="pl4 pr4">
+      <h3>Additional Information</h3>
+      <h4 class="fw2 black-60">Takes 0-10 minutes</h4>
```
Comment (dangerdak) [inline src/views/progress.hbs]: Maybe we should just say it takes 5 minutes so it's consistent with the other sections

File: src/controllers/additional_info.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    })
+    .catch((err) => {
+      console.log(err);
+      req.flash('error', 'Sorry - we couldn\'t load your saved answers for this section');
+      res.render('additional_info', {
+        activePage: { additional_info: true },
+        errorMessages: req.flash('error'),
+        successMessages: req.flash('success'),
+        pageTitle: 'Additional Information',
+        progressPercentage: '80%',
```
Comment (dangerdak) [inline src/controllers/additional_info.js]: The '%' is showing up twice in our progress bar - maybe remove it from here as I think it must be in the progress bar partial. Also the percentage values will need changing on every form page as the form is now longer

Comment (polyccon) [reply src/controllers/additional_info.js]: :+1:

File: src/views/progress.hbs (hunk 1 above)
Comment (polyccon) [reply src/views/progress.hbs]: :+1:

File: src/views/additional_info.hbs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{{> header }}
+{{> progress_bar }}
+
+<section class="mb6">
+{{> messages }}
+  <form class="margin-0-auto w-90 w-60-ns w-80-m mw7" action="/additional_info" method="post">
+  <ol class="pa0">
+    <li class="f5 black-60 db mb2 tracked pt3 pb3"><span class='li mr2 w2'>&rarr;</span>
+      Is there anything else you would like to add?
```
Comment (dangerdak) [inline src/views/additional_info.hbs]: Can you put this in a label tag? then make the "for" attribute of the label equal to the id of the input element

Comment (dangerdak) [reply src/views/additional_info.hbs]: https://www.w3.org/WAI/tutorials/forms/labels/

File: src/controllers/additional_info.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        errorMessages: req.flash('error'),
+        successMessages: req.flash('success'),
+        pageTitle: 'Additional Information',
+        logoutButton: true,
+        progressPercentage: '90',
+        data,
+      });
+    })
+    .catch((err) => {
+      console.log(err);
```
Comment (sohilpandya) [inline src/controllers/additional_info.js]: Might want to remove this console log 😊

**Pull request merged**